spec/forms/teacher_interface/name_and_date_of_birth_form_spec.rb

```diff
@@ -22,7 +22,7 @@
     let(:family_name) { "" }
     let(:date_of_birth) { "" }
 
-    it { is_expected.to be true }
+    it { is_expected.to be false }
```

Comment:
Similar comment to the other form about losing a test for when the form is valid.

Reply:
Looks like we're still missing a test in this file for the valid case.